run-llvm.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+git clone git@github.com:llvm/llvm-project.git
+cd llvm-project
+git checkout remotes/origin/release/15.x
```

Comment:
```suggestion
git clone https://github.com/llvm/llvm-project.git --depth 1 --branch release/15.x
cd llvm-project
```
- 使用https，不需要key
- 指定depth 1和branch，减少下载量